presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1600,4 +1602,39 @@ protocol::NodeStatus PrestoServer::fetchNodeStatus() {
   return nodeStatus;
 }
 
+void PrestoServer::registerDynamicFunctions() {
```

Comment:
@soumiiow @majetideepak : Do you feel we should use a build macro for dynamic functions (PRESTO_ENABLE_DYNAMIC_FUNCTIONS say) like that for remote functions (PRESTO_ENABLE_REMOTE_FUNCTIONS) ? Will definitely make non-IBM teams more safe.